src/js/media-audio-track-menu.ts

```diff
@@ -32,69 +33,73 @@ class MediaAudioTrackMenu extends MediaChromeMenu {
     ];
   }
 
-  #audioTrackList = [];
-  #prevState;
+  #audioTrackList: TextTrackLike[] = [];
+  #prevState: string | undefined;
 
-  attributeChangedCallback(attrName, oldValue, newValue) {
+  attributeChangedCallback(
+    attrName: string,
+    oldValue: string | null,
+    newValue: string | null
+  ): void {
     super.attributeChangedCallback(attrName, oldValue, newValue);
 
     if (
       attrName === MediaUIAttributes.MEDIA_AUDIO_TRACK_ENABLED &&
       oldValue !== newValue
     ) {
-      this.value = newValue;
+      this.value = newValue ?? '';
```

Comment:
this.value == null is checked later on so I think best to leave this null or undefined